Presentation/Program.cs

```diff
@@ -81,7 +84,10 @@
     options.UseNpgsql(builder.Configuration.GetConnectionString("DefaultConnection"));
     options.EnableSensitiveDataLogging();
 });
-builder.Services.AddMapster();
+builder.Services.Configure<CategoryValidationOptions>(builder.Configuration.GetSection("CategoryValidation"));
+builder.Services.AddTransient<IValidator<CategoryDto>, CategoryDtoValidator>();
+builder.Services.Configure<ProductValidationOptions>(builder.Configuration.GetSection("ProductValidation"));
+builder.Services.AddTransient<IValidator<ProductResponseDto>, ProductDtoValidator>();
```

Comment:
first of all, but not that important, order should be adequate.Secondly,Im not sure that its okay to register validators like that and with Transient LifeTime, i will check it, but anyway learn about lifetimes

Reply:
Update:builder.Services.AddValidatorsFromAssembly(Assembly.Load("Application")); that is the best way to register all validator that are or will be in Application project, so change it please to that